spec/requests/sync_spec.rb

```diff
@@ -78,15 +75,12 @@
           allow(JWT::JWK::Set).to receive(:new).with('keys').and_return(jwks)
           allow(JWT).to receive(:decode).with('ABC', nil, true, { algorithms: 'RS256', jwks: jwks }).and_return(decoded)
 
-          allow(AppStoreClient).to receive(:new).and_return(client)
-          allow(client).to receive(:get_submission).and_return(record)
           allow(UpdateSubmission).to receive(:call)
         end
 
         it 'triggers a sync' do
-          post '/app_store_webhook', params: { submission_id: '123' }, headers: { 'Authorization' => 'Bearer ABC' }
+          post '/app_store_webhook', params: { submission_id: '123', data: record }, headers: { 'Authorization' => 'Bearer ABC' }
```

Comment:
do we want to think about the endpoint now that we have chnaged what it does to some extent???